services/QuillLMS/app/workers/canvas_integration/import_teacher_classrooms_students_worker.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module CanvasIntegration
+  class ImportTeacherClassroomsStudentsWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: SidekiqQueue::CRITICAL_EXTERNAL
+
+    PUSHER_EVENT = 'canvas-classroom-students-imported'
+    PUSHER_FAILED_EVENT = 'canvas-account-reauthorization-required'
+
+    def perform(teacher_id, selected_classroom_ids = nil)
+      teacher = ::User.find(teacher_id)
+
```

Comment:
No early return if `teacher.nil?`?  Maybe not necessary, but it's a pattern we've used in a lot of other places.